src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -377,24 +422,36 @@ export default {
 	},
 	apollo: {
 		query: pageQuery,
-		preFetch(config, client, { route }) {
+		preFetch(config, client, { route, cookieStore }) {
 			const publicId = getPublicId(route);
 			return client
 				.query({
 					query: pageQuery,
 					variables: {
 						loanId: Number(route.params?.id ?? 0),
 						publicId,
-						getInviter: !!publicId
+						getInviter: !!publicId,
+						basketId: cookieStore.get('kvbskt')
 					},
 				})
 				.then(({ data }) => {
+					const expCookieSignifier = cookieStore.get('kvlendborrowerbeta');
+					if (expCookieSignifier === 'a' || expCookieSignifier === 'c') {
+						const { query } = route;
+						return Promise.reject({
+							path: `/lend-classic/${route.params.id}`,
+							query,
+						});
+					}
```

Comment:
Maybe in the near future we'll be able to get rid of this expCookie stuff and just provide a query param to allow bypass to the legacy bp...or a link in the page.